content/INTRODUCTION.md

```diff
@@ -0,0 +1,25 @@
+**Requirements**: Have knowledge in Basic PHP programming, Intermediate PHP programming, and Object-oriented Programming in PHP.
+
+**Duration**:
+
+**Standards**: K-12 Computer Science Framework Core Concepts and Practices
+* Algorithms and Programming
+    - Collaborating Around Computing
+    - Recognizing and Defining Computational Problems
+    - Developing and Using Abstractions
+    - Creating Computational Artifacts
+    - Testing and Refining Computational Artifacts
+
+## PHP Advance
+
+PHP advance is designed for:
+
+* Learners who have completed `An Introduction to Basic PHP Programming`, `Intermediate PHP Programming`, and `Object-Oriented Programming (OOP) in PHP`.
+
+* Individuals who have backgrounds in PHP programming.
+
+* Experienced programmers who want to take a refresher.
+
+In this course, you will be stepping into the advance level of PHP. The topics include ternary operator, type hinting, session, cookies, exceptions, and error handling. 
```

Comment:
I think we need to rephrase this statement here since we have to create a new statement on line 15... make sure that the statement here  will serve as the continuation of the above statement. Thanks.